datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -4374,6 +4374,42 @@ GROUP BY dummy
 ----
 text1, text1, text1
 
+# Tests for aggregating with NaN values
+statement ok
+CREATE TABLE float_table (
+    col_f32 FLOAT,
+    col_f32_nan FLOAT,
+    col_f64 DOUBLE,
+    col_f64_nan DOUBLE
+) as VALUES
+( -128.2,  -128.2,                      -128.2, -128.2 ),
+( 32768.3, arrow_cast('NAN','Float32'), 32768.3, 32768.3 ),
+( 27.3,    arrow_cast('NAN','Float64'), 27.3,    27.3 );
```

Comment:
I'm not familiar with this particular corner of SQL but should this be...

```
( -128.2,  -128.2,                      -128.2, -128.2 ),
( 32768.3, arrow_cast('NAN','Float32'), 32768.3, arrow_cast('NAN', 'Float64') ),
( 27.3,    27.3, 27.3,    27.3 );
```

Reply:
Yes, excellent catch -- fixed in 43cb27e669d5ff28f76e70bf54447cbb76df16f2